.github/workflows/dependency_check.yml

```diff
@@ -30,4 +30,4 @@ jobs:
         branch-suffix: short-commit-hash
         base: main
         reviewers: rwedge, thehomebrewnerd, frances-h
-        token: ${{ secrets.REPO_SCOPED_TOKEN }}
+        token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
`GITHUB_TOKEN` should be used by default, can we try dropping this line

Reply:
Done